deploy/kubernetes/overlays/stable/ecr-public/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../bases
+images:
+  - name: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-fsx-csi-driver
```

Comment:
i don't think this works, because the bases https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/deploy/kubernetes/base/node-daemonset.yaml#L36 and https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/deploy/kubernetes/base/controller-deployment.yaml#L34 say amazon/aws-fsx-csi-driver still. It should say `name: amazon/aws-fsx-csi-driver` I think in order to find/replace amazon/aws-fsx-csi-driver with public.ecr.aws/fsx-csi-driver/aws-fsx-csi-driver

Reply:
to be clear this is the diff I mean by "it should say..."
```
-  - ../../bases
+  - ../../../base
 images:
-  - name: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-fsx-csi-driver
-    newName: public.ecr.aws/fsx-csi-driver/aws-fsx-csi-driver
\ No newline at end of file
+  - name: amazon/aws-fsx-csi-driver:v0.9.0
+    newName: public.ecr.aws/fsx-csi-driver/aws-fsx-csi-driver
```